hearts/hearts.py

```
@@ -186,24 +192,13 @@ def add_to_last_trick(self, player, card):
         last_trick = self.tricks[-1]
         last_trick.cards_played.append((player, card))
 
-    def start_round(self, player, card):
-        if card != Card('2', 'c'):
-            raise CardError
-        self.make_new_trick(player, card)
-        self.upkeep(player, card)
-
     def lead_the_trick(self, player, card):
-        try:
-            self.validate_turn(player)
-            self.is_valid_lead(player, card)
+        if self.is_player_turn(player) and self.is_valid_lead(player, card):
             self.make_new_trick(player, card)
             self.upkeep(player, card)
-        except TurnError:
-            # FIXME: Add code to do something later. Reraise the Error to be caught by pytest.
-            raise
-        except HeartsError:
-            # FIXME: Same as above.
-            raise
+        else:
+            pass   # Do something later to deal with out of turn players
```

I would raise an Exception. For now `raise ValueError()`